pallets/amm/Cargo.toml

```diff
@@ -34,6 +34,7 @@ primitives = {path = '../../primitives', default-features = false, version = '2.
 # ORML dependencies
 orml-tokens = {default-features = false, version = "0.3.3-dev"}
 orml-traits = {default-features = false, version = "0.3.3-dev"}
+orml-utilities = {default-features = false, version = "0.3.2"}
```

Comment:
we should use the same 0.3.3-dev. I believe the reason it is not found when you try to specify 0.3.3-dev - is that it is not "patched" in the Cargo.toml in the top level repo dir. If you add the orml-utilitiles to the patch section of that cargo file with the same revision as the other orml crates- you will be able to use 0.3.3-dev version.

Reply:
wrong PR.